src/data/faq.json

```diff
@@ -44,6 +44,17 @@
                                     "FAQ articles on general questions can be found <a href='#ramp_down_general' target='_blank'>here</a>, FAQ articles on implications regarding the app functions <a href='#ramp_down_effects' target='_blank'>here</a> and FAQ articles on alternatives <a href='#ramp_down_alternatives' target='_blank'>here</a>."
                                 ]
                             },
+                            {
+                                "title": "What can I do now? What do I need to know?",
+                                "anchor": "ramp_down_info",
+                                "active": false,
+                                "textblock": [
+                                    "At the end of the Corona-Warn-App development and the cutback of the functions you can still backup your entries and certificates (see also the advice in the <a href='#ramp_down_backup' target='_blank'>FAQ on backup of data</a>).",

```

Comment:
"backup" =>
"back up"

since this is the verb form
https://dictionary.cambridge.org/dictionary/english/back-up

Reply:
Changed that in https://github.com/corona-warn-app/cwa-website/commit/10583e9e6b64094085df38f1651729d1a7544143, thanks!